src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -164,7 +164,7 @@ public async Task ConfigureAsync(CommandSettings command)
                     WriteSection(StringUtil.Loc("EulasSectionHeader"));
 
                     // Verify the EULA exists on disk in the expected location.
-                    string eulaFile = Path.Combine(HostContext.GetDirectory(WellKnownDirectory.Externals), Constants.Path.TeeDirectory, "license.html");
+                    string eulaFile = Path.Combine(HostContext.GetDirectory(WellKnownDirectory.Root), "license.html");
```

Comment:
How will it work if the agent receive auto-update? Will it consider existing file?